G/GStreamer/build_tarballs.jl

```diff
@@ -0,0 +1,56 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "GStreamer"
+version = v"1.18.3"
+
+# Collection of sources required to complete build
+sources = [
+    ArchiveSource("https://gstreamer.freedesktop.org/src/gstreamer/gstreamer-$(version).tar.xz", "0c2e09e18f2df69a99b5cb3bd53c597b3cc2e35cf6c98043bb86a66f3d312100")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir
+apk add bash-completion
+apk add libcap
+apk add gettext
+cd gstreamer-*
+mkdir build
+cd build
+meson .. --cross-file=${MESON_TARGET_TOOLCHAIN}
+ninja
+ninja install
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = supported_platforms()
+
+# The products that we will ensure are always built
+products = [
+    LibraryProduct("libgstcheck-1.0", :libgstcheck),
+    ExecutableProduct("gst-tester-1.0", :gst_tester),
+    LibraryProduct("libgstnet-1.0", :libgstnet),
+    LibraryProduct("libgstreamer-1.0", :libgstreamer),
+    ExecutableProduct("gst-launch-1.0", :gst_launch),
+    ExecutableProduct("gst-stats-1.0", :gst_stats),
+    ExecutableProduct("gst-typefind-1.0", :gst_typefind),
+    LibraryProduct("libgstcontroller-1.0", :libgstcontroller),
+    LibraryProduct("libgstbase-1.0", :libgstbase),
+    ExecutableProduct("gst-inspect-1.0", :gst_inspect)
```

Comment:
https://docs.binarybuilder.org/dev/reference/#BinaryBuilderBase.LibraryProduct
```julia
julia> using Base.BinaryPlatforms

julia> parse_dl_name_version("libgstcheck-1.0.dylib", "macos")[1]
"libgstcheck-1"
```
so you have to do
```julia
    LibraryProduct(["libgstcheck-1.0", "libgstcheck-1"], :libgstcheck),
```
and so on...